client/jetpack-connect/example-components/jetpack-connect.jsx

```diff
@@ -45,14 +45,14 @@ const JetpackConnectExampleConnect = ( { isLegacy, url, translate, onClick } ) =
 					<div className="example-components__content-wp-admin-connect-banner">
 						{ ! isLegacy ? (
 							<div className="example-components__content-wp-admin-plugin-name" aria-hidden="true">
-								{ translate( 'Your Jetpack is almost ready!', {
+								{ translate( 'Connect Jetpack to WordPress.com', {
 									context:
 										'Jetpack Connect activate plugin instructions, connection banner headline',
 								} ) }
 							</div>
 						) : null }
 						<div className="example-components__content-wp-admin-connect-button" aria-hidden="true">
-							{ translate( 'Connect to WordPress.com', {
+							{ translate( 'Set up Jetpack', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?